### PR TITLE
techpass rbac - shared responsibility and dos and don't of techpass

### PR DESCRIPTION
**Pull request by yilunyou**

### CODE REVIEW

File: docs/concepts/rbac.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+- **What** actions can they perform on our application/service?
+
+## The Who?
+Managed using TechPass.
+
+There are 2 types of Coarse Grain access controls that you can setup to allow users access to your application/services.
+- individual users
+- groups
+
+These are achieved using [User Assignment](applications/#user-assignment-required) feature
```
Comment (jovanhyw) [inline docs/concepts/rbac.md:25]: the link leads to a 404, removing the `/` will fix it. `applications#user-assignment-required`

File: docs/concepts/rbac.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+There are 2 types of Coarse Grain access controls that you can setup to allow users access to your application/services.
+- individual users
+- groups
+
+These are achieved using [User Assignment](applications/#user-assignment-required) feature 
+
+### User Assignment
+In TechPass, Tenants can configure their applications to only allow authorised users to sign in. 
+This is done by ensuring the [*User Assignment Required*](applications/#user-assignment-required) setting is checked.
```
Comment (jovanhyw) [inline docs/concepts/rbac.md:29]: here too

File: docs/concepts/rbac.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+These are achieved using [User Assignment](applications/#user-assignment-required) feature 
+
+### User Assignment
+In TechPass, Tenants can configure their applications to only allow authorised users to sign in. 
+This is done by ensuring the [*User Assignment Required*](applications/#user-assignment-required) setting is checked. 
+With this setting enabled, any user who wishes to sign in to your application will need to be assigned access to the app. This allows the tenant to perform pre-approval checks on the user before access is granted. 
+This is useful for application that are/should be accessible only to a specific subset of TechPass users. 
+
+?> Assigning access to users can be programmatically given via TechPass's [*Automation APIs*](apis). The APIs provide tenants with the versatility to implement automated pre-approval checks in their backend and then call the relevant API to assign access once the checks are completed.
```
Comment (jovanhyw) [inline docs/concepts/rbac.md:33]: link leads to 404 here also `apis/_sidebar` or `apis/integration`

File: docs/concepts/rbac.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  You can choose to use groups as a logical representation of your different users.
+  Eg. group by agencies/teams. 
+- **Security/Role**:   
+  Or you can use groups to provide access controls to different resources within your service.   
+  Eg. A billing group to access finance & billing service. Add project manager and finance executives to that group. 
+  A admin group to access all features or audit group to access only the logs and reporting, etc
+  
+The tenant admins can carry out their own verification of actual user identity before granting access.
+
+Off-the-shelf proprietary applications should already come with some sort of access control mechanism built-in. Depending on the capability of the application, it may not be possible for tenants to manage permissions via TechPass. Tenants will need manage access locally on these applications using the built-in feature.
```
Comment (jovanhyw) [inline docs/concepts/rbac.md:52]: last sentence missing a word, `Tenants will need to manage...`

File: docs/concepts/rbac.md (hunk 3 above)
Comment (yilunyou) [reply docs/concepts/rbac.md:33]: good catch, fixed

File: docs/concepts/rbac.md (hunk 4 above)
Comment (yilunyou) [reply docs/concepts/rbac.md:52]: another catch! thanks

File: docs/concepts/rbac.md (hunk 2 above)
Comment (yilunyou) [reply docs/concepts/rbac.md:29]: oo.. this one i need to test

File: docs/concepts/rbac.md (hunk 1 above)
Comment (yilunyou) [reply docs/concepts/rbac.md:25]: this one also

**Review by yang-lijia — APPROVED** [review]

**Review by jovanhyw — APPROVED** [review]

**Pull request merged**